docs/Get-HelpPreview.md

```diff
@@ -5,25 +5,25 @@ schema: 2.0.0
 
 # Get-HelpPreview
 ## SYNOPSIS
-Preview the output Get-Help would return from an external help file(s).
+Displays your generated external help as **Get-Help** output.
+
 ## SYNTAX
 
 ```
-Get-HelpPreview -Path <String[]> [<CommonParameters>]
```

Comment:
I'm surprised I made a change like that - I don't remember adding CommonParameters to anything.